nilearn/input_data/tests/test_nifti_masker.py

```diff
@@ -203,7 +206,10 @@ def test_5d():
     # Test that, in list of 4d images with last dimension=1, they are
     # considered as 3d
 
-    data_5d = [np.random.random((10, 10, 10, 3)) for i in range(5)]
+    data_5d = [
+        np.random.RandomState(42).random_sample((10, 10, 10, 3))
```

Comment:
Why not creating the generator outside of the loop ?

Reply:
note that as it is, it creates 5 times the same data (the seed is reset every time)

Reply:
Ahha good catch, thanks. Will change it.